### PR TITLE
Versioning Strategy

### PR DESCRIPTION
**Pull request by FrankEssenberger**
Please provide a description of what your change does and why it is needed.

Closes SAP/cloud-sdk-backlog#19

This is a first discussion basis for the meeting and will be polished after discussion.

### CODE REVIEW

File: knowledge-base/adr/0027-versioning-strategy.md [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+
+For the clients the situation is much less coupled than for core.
+Some services will update every two weeks (workflow) other every year (S/4 OnPremise).
+We will do versioning in the following way:
+
+- Major versions of clients aligns with major version of core.
+- Core versions are not fixed in the clients -> installation updates core to latest.
+- Pipeline generates the clients and detects changes.
+- A change in the clients relates to a minor version increase.
+- Pipeline runs every two weeks and publishes only the changed clients with increased minor version.
```
Comment (jjtang1985) [inline knowledge-base/adr/0027-versioning-strategy.md:40]: Does it mean we need the release processes for every two weeks with Karsten?

Comment (FrankEssenberger) [reply knowledge-base/adr/0027-versioning-strategy.md:40]: Yes, but the java team does it as well and have some sort of optimized sirius flow

Comment (jjtang1985) [reply knowledge-base/adr/0027-versioning-strategy.md:40]: Ok, let's try it then.

Comment (FrankEssenberger) [reply knowledge-base/adr/0027-versioning-strategy.md:40]: +1

**Pull request merged**